readinglist/views/article.py

```diff
@@ -14,14 +14,29 @@ def preparer(self, appstruct):
         return strip_whitespace(appstruct)
 
 
+class URL(SchemaNode):
+    """String representing a URL."""
+    schema_type = String
+    validator = colander.All(colander.url, colander.Length(min=1, max=2048))
```

Comment:
this question could be handled in a separate issue, since it also concerns article titles (@rnewman proposition)